src/main/app/case/to-api-format.ts

```diff
@@ -87,6 +87,7 @@ const fields: ToApiConverters = {
     ...(data.applicationType === ApplicationType.JOINT_APPLICATION
       ? setUnreachableAnswersToNull([
           'applicant1IsApplicant2Represented',
+          'applicant2SolicitorRepresented',
```

Comment:
I missed this in my changes though, so should keep this in

Reply:
@aramchurn21 can you update the pr to keep this line and the relevant test in please - it's something I missed in my one

Reply:
Ok thanks will do.